### ISSUE TITLE
brew-file get_files is broken

### ISSUE THREAD

**Issue by zcutlip** [rcmdnk/homebrew-file#156]
It appears a `pathlib.Path` object is being used where a string is expected:

```python
Traceback (most recent call last):
  File "/opt/homebrew/bin/brew-file", line 3556, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/homebrew/bin/brew-file", line 3543, in main
    b.execute()
  File "/opt/homebrew/bin/brew-file", line 2886, in execute
    self.get_files(is_print=True, all_files=self.opt["all_files"])
  File "/opt/homebrew/bin/brew-file", line 2127, in get_files
    self.log.info("\n".join(files))
                  ^^^^^^^^^^^^^^^^

TypeError: sequence item 0: expected str instance, PosixPath found
```

**Comment by rcmdnk** [comment]
Thanks for the report!
It should be fixed at v9.0.7. Could you please update brew-file and try it?

**Comment by zcutlip** [comment]
Fixed. Thank you!
